x-pack/solutions/security/plugins/security_solution/public/management/components/endpoint_execute_action/execute_action_host_response.tsx

```diff
@@ -64,16 +64,18 @@ export const ExecuteActionHostResponse = memo<ExecuteActionHostResponseProps>(
               textSize={textSize}
               showPasscode={showPasscode}
             />
-            <EuiSpacer size="xxl" />
```

Comment:
This was adjusted to make spacing consistent in size and not render spacing when no output was present